.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,37 @@
+settings:
+    # Jira project key to create the issue in
+  jira_project_key: "IAM"
+
+  # Dictionary mapping GitHub issue status to Jira issue status
+  status_mapping:
+    opened: Untriaged
+    closed: done
+
+  # (Optional) Jira project components that should be attached to the created issue
+  # Component names are case-sensitive
+  components:
+    - Kratos
```

Comment:
```suggestion
    - Login UI
```

Reply:
That's from another branch, it was just showing because I force pushed main and didnt rebase this branch. It's gone now.